velox/dwio/parquet/duckdb_reader/ParquetReader.cpp

```diff
@@ -321,7 +321,12 @@ ParquetReader::ParquetReader(
   std::vector<TypePtr> types;
   types.reserve(reader_->return_types.size());
   for (auto& t : reader_->return_types) {
-    types.emplace_back(duckdb::toVeloxType(t));
+    types.emplace_back(duckdb::toVeloxType(t, options.isReadAsLowerCase()));
```

Comment:
DuckDB's Parquet reader is no longer maintained. Let's remove this change and changes to DuckConversion.*. We should delete DuckDB's Parquet reader.